src/console_op/k8s.go

```diff
@@ -216,3 +229,16 @@ func updateNodesPerPod(newNumMtn, newNumRvr int) {
 	numMtnNodesPerPod = newNumMtn
 	numRvrNodesPerPod = newNumRvr
 }
+
+// Find and return where the current pod is running in k8s
+func (K8Manager) getPodLocation(podID string) (loc string, err error) {
```

Comment:
Is this returning xname, nid, or ncn-w??? alias? It would be good to specify as xname is what we really need when looking at console scheduling.